playwright/specs/account.spec.ts

```diff
@@ -0,0 +1,38 @@
+import test from "../fixtures/common";
+import { expect } from "@playwright/test";
+import { AccountModal } from "../models/AccountModal";
+import { DeviceAction } from "../models/DeviceAction";
+
+// specific environment
+test.use({ userdata: "skip-onboarding" });
+
+const currencies = ["BTC", "LTC", "ATOM"];
+
+test.describe.parallel("Accounts", () => {
+  for (const currency of currencies) {
+    test(`[${currency}] Add account`, async ({ page }) => {
+      const accountModal = new AccountModal(page);
+      const deviceAction = new DeviceAction(page);
+
+      await test.step(`[${currency}] Open modal`, async () => {
+        await accountModal.open();
+        expect(await page.screenshot()).toMatchSnapshot(`open-modal.png`);
```

Comment:
We seem to be going in the direction of doing all our asserts as Matching Snapshots. I like this since it means we are checking for styling regressions at the same time as functional tests, but I wonder if test runs will be flakier and slower as a result of all the screenshots. Also do we need to worry about storing lots of image files or anything?

I'm ok with it as it is but just want to make sure we are happy with this approach going forward